### PR TITLE
Added instruction about how to verify CoreOS images

### PR DESCRIPTION
**Pull request by kayrus**
@robszumski @brianredbeard

### CODE REVIEW

**Comment by kayrus** [comment]
Don't know where exactly should I put this info...

**Comment by robszumski** [comment]
Let's put this under `cluster-management/setup/verify-images/index.md`.

I think we need some background info at the top of the doc that gives an overview of why and how we sign images, and explain where they are stored, etc.

File: running-coreos/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Please follow the instruction below to know how to verify CoreOS images.
```
Comment (robszumski) [inline running-coreos/index.md]: We need to add the front matter for jekyll and a title

Comment (robszumski) [inline running-coreos/index.md]:
```
---
layout: docs
title: Verify CoreOS Images with GPG
category: cluster_management
sub_category: setting_up
fork_url: https://github.com/coreos/docs/blob/master/cluster-management/setup/<INSERT>
weight: 7
---
```

Comment (robszumski) [inline running-coreos/index.md]:
```
# Verify CoreOS Images with GPG
```

Comment (robszumski) [inline running-coreos/index.md]: That was my initial idea for a title, feel free to pitch changes as well.

File: running-coreos/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (robszumski) [inline running-coreos/index.md]: First stab at an overview section:

CoreOS publishes new images for each release across a variety of platforms and hosting providers. Each channel has it's own set of images ([stable](http://stable.release.core-os.net/amd64-usr/current/), [beta](http://beta.release.core-os.net/amd64-usr/current/), [alpha](http://alpha.release.core-os.net/amd64-usr/current/)) that are posted to our storage site. Along with each image, a signature is generated from the [CoreOS Image Signing Key](https://coreos.com/security/image-signing-key/) and posted.

After downloading your image, you should verify it with `gpg`. First, download the image signing key:

File: running-coreos/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+First of all you have to install `gpg` tool then you have to download public image signing key:
+
+```sh
+curl -O https://coreos.com/security/image-signing-key/CoreOS_Image_Signing_Key.asc
+```
+
+Import public key and verify its ID with one on https://coreos.com/security/image-signing-key/ page:
```
Comment (robszumski) [inline running-coreos/index.md]: Next, import the public key and verify that the ID matches the website:

File: running-coreos/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+```sh
+gpg --import --keyid-format LONG CoreOS_Image_Signing_Key.asc
+gpg: key 50E0885593D2DCB4: public key "CoreOS Buildbot (Offical Builds) <buildbot@coreos.com>" imported
+gpg: Total number processed: 1
+gpg:               imported: 1  (RSA: 1)
+gpg: 3 marginal(s) needed, 1 complete(s) needed, PGP trust model
+gpg: depth: 0  valid:   2  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 2u
+```
+
+Download CoreOS image and GPG signature, i.e. QEMU production image:
```
Comment (robszumski) [inline running-coreos/index.md]: Now we're ready to download an image and it's signature, ending in `.sig`. We're using the QEMU image in this example:

File: running-coreos/index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Verify image with `gpg` tool:
+
+```sh
+gpg --verify coreos_production_qemu_image.img.bz2.sig
+gpg: Signature made Tue Jun 23 09:39:04 2015 CEST using RSA key ID E5676EFC
+gpg: Good signature from "CoreOS Buildbot (Offical Builds) <buildbot@coreos.com>"
+```
+
+The `Good signature` message indicates that the file signature is valid.
```
Comment (robszumski) [inline running-coreos/index.md]: The `Good signature` message indicates that the file signature is valid. Go launch some machines now that we've successfully verified that this CoreOS image isn't corrupt, that it was authored by CoreOS, and wasn't tampered with in transit.

**Comment by kayrus** [comment]
Done

**Comment by robszumski** [comment]
LGTM

**Pull request merged**